sdk/ml/azure-ai-ml/tests/dsl/unittests/test_controlflow_pipeline.py

```diff
@@ -102,7 +97,7 @@ def invalid_pipeline():
                     # invalid JSON string items
 
                     '[{"component_in_number": 1}, {}]',
-                    "Items has to be list/dict of non-empty dict as value"
+                    "Items should to have same keys with body inputs"
```

Comment:
Nit: "Items should have same keys with body inputs" (no **to**), here and `parallel_for.py`